core/sql/generator/GenRelUpdate.cpp

```diff
@@ -2855,8 +2859,8 @@ short HbaseInsert::codeGen(Generator *generator)
            setVsbbInsert(TRUE);
       }
 
-      if ((isUpsert()) &&
-	  (getInsertType() == Insert::UPSERT_LOAD))
+      
+        if (getInsertType() == Insert::UPSERT_LOAD)
```

Comment:
Just to make sure I understand: You removed the isUpsert() here because you know that for upserts that are VSBB inserts, the insert type is UPSERT_LOAD?